dev/src/dev/migrate.clj

```diff
@@ -116,27 +118,39 @@
 (defn- change->sql
   [change sql-generator-factory database]
   {:forward  (stmts-to-sql (.generateStatements change database) sql-generator-factory database)
-   :rollback (stmts-to-sql (.generateRollbackStatements change database) sql-generator-factory database)})
-
-(defn migration-sql-by-id
-  "Get the sql statements for a specific migration ID.
-    (migration-sql-by-id \"v51.2024-06-12T18:53:02\")
+   :rollback (try (stmts-to-sql (.generateRollbackStatements change database) sql-generator-factory database)
+                  (catch RollbackImpossibleException e
+                    (str "Rollback impossible " e)))})
```

Comment:
Instead of just getting an error on a migration without a rollback:
```
(dev/migration-sql-by-id "v49.2024-06-27T00:00:05" :postgres)
; Execution error (RollbackImpossibleException) at liquibase.change.AbstractChange/generateRollbackStatementsFromInverse (AbstractChange.java:553).
; No inverse to liquibase.change.core.DropUniqueConstraintChange created
```
You now get a useful result and an error message
```
{:forward "ALTER TABLE metabase_field DROP CONSTRAINT idx_uniq_field_table_id_parent_id_name;",
 :rollback
 "Rollback impossible liquibase.exception.RollbackImpossibleException: No inverse to liquibase.change.core.DropUniqueConstraintChange created"}
```

I'm not sure why `generateRollbackStatements` results in this error in this case, since `49.2024-06-27T00:00:05` does have a rollback defined. But I'm not going to investigate that right now and this change is useful regardless